src/Features/OnboardingTasks/Tasks/AdditionalPayments.php

```diff
@@ -73,7 +73,7 @@ public function can_view() {
 
 		$woocommerce_payments = new WooCommercePayments();
 
-		if ( $woocommerce_payments->is_requested() && $woocommerce_payments->is_supported() && ! $woocommerce_payments->is_connected() ) {
+		if ( ! $woocommerce_payments->is_requested() || ! $woocommerce_payments->is_supported() || ! $woocommerce_payments->is_connected() ) {
 			// Hide task if WC Pay is installed via OBW, in supported country, but not connected.
```

Comment:
I think this comment just needs to be updated to match the `if` conditions.

Reply:
Nice catch @adrianduffell! I fixed it in commit [be63abf](https://github.com/woocommerce/woocommerce-admin/pull/8506/commits/be63abf3c5e2ab44e0da7ea415b3cc7cf6888d3f)